AppRouter.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import Relay from 'react-relay';
+import {
+    applyRouterMiddleware,
+    IndexRoute,
+    Link,
+    Route,
+    Router,
+    browserHistory
+} from 'react-router';
+import useRelay from "react-router-relay";
+
+import HomePage from './pages/HomePage';
+
+let baseUrl = '/example';
+const { hostname, pathname } = window.location;
+
+if (hostname === 'localhost') {
+  try{
+      baseUrl = pathname.match(/^\/project\/([a-f0-9]+\-)+[a-f0-9]+\/?/)[0];
+  } catch(e) {}
+}
+
+export default (App) => {
+    return (
+        <Router
+            environment={Relay.Store}
+            history={browserHistory}
```

Comment:
Pls, check with @agutoli if we'll keep using react's history. I think it would change